test/common/user_permissions.go

```diff
@@ -0,0 +1,227 @@
+package common
+
+import (
+	goctx "context"
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/google/go-querystring/query"
+	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/test/resources"
+	projectv1 "github.com/openshift/api/project/v1"
+	v1 "github.com/openshift/api/route/v1"
+	corev1 "k8s.io/api/core/v1"
+	"strings"
+
+	"io/ioutil"
+	"k8s.io/apimachinery/pkg/types"
+	"net/http"
+	"testing"
+)
+
+const (
+	openshiftAuthenticationNamespace = "openshift-authentication"
+	openshiftOAuthRouteName          = "oauth-openshift"
+
+	pathProjects = "api/kubernetes/apis/project.openshift.io/v1/projects"
+	pathFusePods = "api/kubernetes/api/v1/namespaces/redhat-rhmi-fuse/pods"
+
+	expectedRhmiDeveloperNamespace    = "redhat-rhmi-fuse"
+	expectedRhmiDeveloperProjectCount = 1
+	expectedFusePodCount              = 6
+)
+
+// struct used to create query string for fuse logs endpoint
+type LogOptions struct {
+	Container string `url:"container"`
+	Follow    string `url:"follow"`
+	TailLines string `url:"tailLines"`
+}
+
+func TestIntegreatlyUserPermissions(t *testing.T, ctx *TestingContext) {
+	// get the RHMI custom resource to check what storage type is being used
+	rhmi := &v1alpha1.RHMI{}
+	ns := fmt.Sprintf("%soperator", namespacePrefix)
+	if err := ctx.Client.Get(goctx.TODO(), types.NamespacedName{Name: installationName, Namespace: ns}, rhmi); err != nil {
+		t.Fatal("error getting RHMI CR:", err)
+	}
```

Comment:
There is a shared function to retrieve this info:
https://github.com/integr8ly/integreatly-operator/blob/edcbe34d3b2b94b93573d5236b85e43e69670697/test/common/shared_functions.go#L75